conf/gce.yaml.template

```diff
@@ -1,12 +1,8 @@
 GCE:
   # Google Provider as Compute Resource
-  # Project name in Google provider
-  PROJECT_ID:  # example-project-id
-  # Service Account email id which has compute admin permission
-  CLIENT_EMAIL: client@example.com
   # client json Certificate path which is local path on satellite
   CERT_PATH: /path/to/certificate.json
   # Zones
   ZONE: example-zone
-  # client certificate URL
-  CERT_URL: /path/to/certificate/url
+  # client certificate
+  CERT: {} # client json Certificate
```

Comment:
I believe this is causing problems in test collection in the github action, where these template files are used.

Reply:
Are you sure those templates files are used? Looks like no matter what I put in this field test collection fails. I tried also blank field, empty string, random string and collection is still failing. Maybe I can use `settings.gce.cert or ""` in test but this will be workaround and won't really fix issue